packages/quantic/force-app/main/default/lwc/quanticResultActionBar/quanticResultActionBar.js

```diff
@@ -0,0 +1,83 @@
+import {LightningElement} from 'lwc';
+
+/**
+ *  The `QuanticResultActionBar` component displays multiple result actions in a proper way and with the right styles.
+ * @category Search
+ * @category Insight Panel
+ * @example
+ * <c-quantic-result-action-bar>
+ *   <c-quantic-result-action onclick={actionOne} icon-name="utility:attach" label-when-hover="Action One"></c-quantic-result-action>
```

Comment:
the prop names for `c-quantic-result-action` might still change but they're also a bit out of sync with what's currently in the PR.